src/features/home-page/date-time/show-date.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+class ShowDate extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: new Date().toDateString(),
+    };
+  }
+  render() {
+    return <div className="Data">{this.state.date}</div>;
+  }
+}
+
+export default ShowDate;
```

Comment:
@IraLukianovets where do you use this Component?

Reply:
@StadnykYura I need an extra space on the navbar. If to put both components in only one navbar cell it will look ugly. As well as, the font will be too tiny.

Reply:
@IraLukianovets  You can make time component wider than other bars, if you need it. And i think that you don't need 2 components for time.